vault/login_mfa.go

```diff
@@ -2653,6 +2653,31 @@ func (b *LoginMFABackend) MemDBMFALoginEnforcementConfigByNameAndNamespace(name,
 	return eConfig.Clone()
 }
 
+func (b *LoginMFABackend) MemDBMFALoginEnforcementConfigByID(id string) (*mfa.MFAEnforcementConfig, error) {
```

Comment:
Does it make sense to add unit tests? Not sure where exactly this is being used to add more integration like tests.

Reply:
We don't have unit tests for the other associated methods. If we were to implement them we'd likely need a core anyway. The functionality this method is implementing is only used in one area which has an accompanying enterprise test.